cli/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "create-ol-kit-map",
+  "version": "1.0.0",
+  "team": "Bayer",
+  "homepage": "https://github.com/MonsantoCo/ol-kit",
+  "description": "",
+  "keywords": [
+    "ol-kit",
+    "open-source-map"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MonsantoCo/ol-kit"
+  },
+  "dependencies": {
+    "child-process-promise": "^2.2.1",
+    "depcheck": "^1.2.0",
+    "vorpal": "^1.12.0"
+  },
+  "scripts": {
+    "create-map": "node index.js"
```

Comment:
try this: `npx i @bayer/ol-kit@next create-map` (we can release a beta version to test this locally)

Reply:
bump `~/package.json` version to `1.1.0-0`, use `npm run ship`, run command above^